### PR TITLE
Remove All Weapons

### PR DESCRIPTION
**Pull request by robbybaseplate**

### CODE REVIEW

File: ac_config.lua [hunk 1]
```diff
@@ -19,6 +19,8 @@ Components = {
 	["client.duiblacklist"] = true, -- checks for certain runtime textures being created which are used in cheat menus
 	["client.bypasshacks"] = true, -- checks if their cheats try to disable certain anticheat components, not exclusive to anticheese
 	["client.esx.gcphone"] = true, -- various detections for gcphone crashers/abuse
+	["client.night"] = true, -- anti night vision
```
Comment (Blumlaut) [inline ac_config.lua:22]: any reason this was moved between some ESX settings? would probably better off being before or after them.

Comment (robbybaseplate) [reply ac_config.lua:22]: to be with client.

Comment (Blumlaut) [reply ac_config.lua:22]: would probably best moving this above or below esx, just so its not "somewhere in the middle"

Comment (robbybaseplate) [reply ac_config.lua:21]: Changed

**Pull request merged**